cli/commands/service/discovery/discovery.go

```diff
@@ -87,6 +87,14 @@ func NewQueryDiscoveryCommand() *cobra.Command {
 
 			client = discovery.NewDiscoveryClient(session)
 
+			req = discovery.ListResourcesRequest{
```

Comment:
We also need to pass this `req` in line 102 as the first parameter to `api.ListAllPaginated`